### ISSUE TITLE
ChoiceGroup: focus not visible on keyboard access

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#1716]
#### Description:
There is no visual feedback when keyboard focus is present on choice group component.

#### Repro:
http://dev.office.com/fabric#/components/choicegroup

#### Actual behavior:
When focus is on the component
![image](https://cloud.githubusercontent.com/assets/11937034/25691012/8999e5f6-30b4-11e7-9aaa-8dfbb4169cc0.png)

#### Expected behavior:
A dotted border should be present to highlight keyboard focus.

#### Details:
The div and corresponding css are as below:

![image](https://cloud.githubusercontent.com/assets/11937034/25691009/7ff48916-30b4-11e7-8d9a-aecf2085b1f7.png)

![image](https://cloud.githubusercontent.com/assets/11937034/25691019/9e337734-30b4-11e7-9ea0-50954b6bced1.png)

Related issue #1115 
Currently, HC also does not show any border

**Comment by betrue-final-final** [comment]
Our Fabric focus is solid neutralSecondary, and not dotted.

**Comment by micahgodbolt** [comment]
![image](https://cloud.githubusercontent.com/assets/1434956/25765407/ca3d30c0-31a1-11e7-855b-fae55ad31f81.png)


![image](https://cloud.githubusercontent.com/assets/1434956/25765389/b1e1b23a-31a1-11e7-9abe-614eb66a465c.png)

![image](https://cloud.githubusercontent.com/assets/1434956/25765398/bd2e545e-31a1-11e7-9ff1-bf3d83b8c145.png)
